src/pages/home/Employee/EmployeeHome.js

```diff
@@ -0,0 +1,186 @@
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+
+    if (!pickupLocation) {
+      setPickupLocationErrorText("Please enter pickup location");
+    } else {
+      setPickupLocationErrorText("");
+    }
+    if (!projectCode) {
+      setProjectCodeErrorText("Please enter project code");
+    } else {
+      setProjectCodeErrorText("");
+    }
+    if (!dropLocation) {
+      setDropLocationErrorText("Please enter drop location");
+    } else {
+      setDropLocationErrorText("");
+    }
+  };
+  const handleTabChange = (event, newValue) => {
+    setTabIndex(newValue);
+  };
+  return (
+    <>
+      <Stack direction={"row"} spacing={2}>
+         <img src={logoImage} className="logo"/>
+         <Box
+              sx={{
+                display: "flex",
+                justifyContent: "flex-end",
+                marginTop: 4,
+                marginRight: 4,
+                marginLeft: "auto"
+              }}
+         >
+           <Avatar imageLink={user?.picture} />
+         </Box>
+      </Stack>
+      <h4 className="welcomeMessage">{msg}</h4>
+        <Box sx={{marginLeft: "90px", marginRight: "90px"}}>
+            <Accordion>
+                <AccordionSummary
+                    expandIcon={<ExpandMoreIcon />}
+                    aria-controls="panel1a-content"
+                    id="panel1a-header"
+                    onClick={() => setOpen(!isOpen)}
+                >
+                    <Typography>
+                        {isOpen
+                          ? "Planning an upcoming travel, create your travel request here"
+                          : "Fill the form to create a new travel request"}
+                    </Typography>
+                </AccordionSummary>
+                <AccordionDetails onFocus={() => setOpen(false)}>
+                    <Typography>
+                        <FormControl className="form-control-container">
+                            <div>
+                                <TextField
+                                    sx={{marginRight: "65px"}}
+                                    variant="standard"
+                                    type="text"
+                                    required
+                                    placeholder="Project Code"
+                                    id="projectCode"
+                                    name="projectCode"
+                                    className="text-field"
+                                    value={projectCode}
+                                    error={!!projectCodeErrorText}
+                                    helperText={projectCodeErrorText}
+                                    onChange={(e) => setProjectCode(e.target.value)}
+                                />
+                                <DatePicker placeholderText="Select a date and time" selected={startDate} onChange={(date) => setStartDate(date)} dateFormat="Pp" showTimeSelect timeFormat="p" />
```

Comment:
The date picker should block us from picking the date values from previous or past days we can check on this and customise it.